src/nupic/utils/LogItem.cpp

```diff
@@ -66,7 +66,7 @@ LogItem::~LogItem()
   }
 
   if (ostream_ == nullptr)
-    ostream_ = &(std::cout);
+    ostream_ = &(std::cerr);
```

Comment:
Libraries should not be dumping diagnostic info to stdout, since this will conflate the data stream of processes communicating via pipeline.